README.md

```diff
@@ -70,8 +71,11 @@ your own repository.
     - Staging: `<PROJECT-NAME>-staging`
     - Production: `<PROJECT-NAME>-production`
 3. Turn on "Review Apps" from the Pipeline's page
-4. Set `MAIL_FROM` in your Heroku config vars (review, staging and production).
-5. Set `DOMAIN_NAME` in your Heroku config vars (staging and production), links in email will point to the provided domain name.
+4. Set Heroku config vars:
+   - `SEED_ADMIN_EMAIL` (review, staging and production)
```

Comment:
production and seeds?

Reply:
uhhh you are right 🤔

Reply:
Currently the master branch runs the seeds in the production env, so this content is correct.
It is however going to be fixed shortly :-)